TombEngine/Objects/Effects/Boss.cpp

```diff
@@ -212,6 +212,12 @@ namespace TEN::Effects::Boss
 			{
 				auto pos = Random::GeneratePointInSphere(sphere);
 				SpawnExplosionSmoke(pos);
+
+				TriggerExplosionSparks(
+					item.Pose.Position.x + (GetRandomControl() % 128 - 64 * 2),
```

Comment:
Do you know why it's GetRandomControl() % 128 - 64 * 2 ? could you change it with Random::GenerateInt/Float ?

Reply:
fixed